integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/call/CallMediatorBlockingSenderTestCase.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.wso2.carbon.esb.mediator.test.call;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+
+import java.io.File;
+import java.net.URL;
+
+import static org.wso2.carbon.esb.mediator.test.call.Util.executeAndAssert;
+
+public class CallMediatorBlockingSenderTestCase extends ESBIntegrationTest {
+
+    @BeforeClass(alwaysRun = true)
+    public void deployService() throws Exception {
+        super.init();
+        String[] configs = {"HTTPStatusResponseAPI.xml", "AcceptedEndpointProxy.xml", "BadRequestEndpointProxy.xml",
+                "ConflictEndpointProxy.xml", "ForbiddenEndpointProxy.xml", "NotFoundEndpointProxy.xml",
+                "ServerErrorEndpointProxy.xml", "UnauthorizedEndpointProxy.xml"};
+        for (String configName : configs) {
+            loadESBConfigurationFromClasspath(File.separator + "artifacts" + File.separator + "ESB"
+                    + File.separator + "mediatorconfig" + File.separator + "call_blocking" + File.separator
+                    + configName);
+        }
+        verifyAPIExistence("HTTPStatusResponseAPI");
```

Comment:
Shall we verify the existence of other configs also ( proxies )

Reply:
Done